### PR TITLE
Oleg refactor

### PR DESCRIPTION
**Pull request by orovinskiy**
refactored checklist and implemented kellers new style

### CODE REVIEW

File: model/Database.php [hunk 1]
```diff
@@ -334,6 +336,52 @@ function getTutorsChecklist($year, $userID)
         return $results;
     }
 
+    function getTutorName($userID){
```
Comment (kellerflint) [inline model/Database.php:346]: Missing PHP Docs

File: model/Database.php [hunk 2]
```diff
@@ -334,6 +336,52 @@ function getTutorsChecklist($year, $userID)
...
+        $statement = $this->_dbh->prepare($sql);
+
+        $statement->execute([$userID]);
+
+        $results = $statement->fetchAll(PDO::FETCH_ASSOC);
+
+        return $results[0]['tutor_first'].' '.$results[0]['tutor_last'];
+    }
+
+    function getNextState($stateID){
```
Comment (kellerflint) [inline model/Database.php:364]: Missing PHP Docs

File: model/Database.php [hunk 3]
```diff
@@ -334,6 +336,52 @@ function getTutorsChecklist($year, $userID)
...
+        $statement = $this->_dbh->prepare($sql);
+
+        $statement->execute([$stateID]);
+
+        $results = $statement->fetchAll(PDO::FETCH_ASSOC);
+
+        return $results[0]['state_set_by'];
+    }
+
+    function getNextStateText($stateID){
```
Comment (kellerflint) [inline model/Database.php:383]: Missing PHP Docs

File: model/Database.php [hunk 4]
```diff
@@ -334,6 +336,52 @@ function getTutorsChecklist($year, $userID)
...
+        $statement = $this->_dbh->prepare($sql);
+
+        $statement->execute([$stateID]);
+
+        $results = $statement->fetchAll(PDO::FETCH_ASSOC);
+
+        return $results[0]['state_text'];
+    }
+
+    function updateStateOfTutor($state,$item,$user){
```
Comment (kellerflint) [inline model/Database.php:402]: Missing PHP Docs

**Review by kellerflint — CHANGES_REQUESTED** [review]
Code looks good, you got that done fast! You're missing PHP docs on the new database functions. Also, I didn't see any deleted functions in the Database which surprised me. Were none of your previous functions for this page made obsolete? If they are no longer being used, you might consider deleting them so we can cut down on the size of this class.

File: model/Database.php (hunk 2 above)
Comment (orovinskiy) [reply model/Database.php:364]: all comments have been added

**Review by kellerflint — APPROVED** [review]
Great!

**Pull request merged**